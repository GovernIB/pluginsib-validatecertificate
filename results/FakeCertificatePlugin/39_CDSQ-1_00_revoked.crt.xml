<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<resultatValidacio>
    <informacioCertificat>
        <altresValors/>
        <classificacio>-1</classificacio>
        <emissorID>CN=EC-SectorPublic,OU=Serveis Públics de Certificació,O=CONSORCI ADMINISTRACIO OBERTA DE CATALUNYA,C=ES</emissorID>
        <emissorOrganitzacio>CONSORCI ADMINISTRACIO OBERTA DE CATALUNYA</emissorOrganitzacio>
        <entitatSubscriptoraNom>Organització de prova</entitatSubscriptoraNom>
        <nomDomini>webtest.aoc.cat</nomDomini>
        <numeroSerie>3261199091937075586</numeroSerie>
        <oiEuropeu>VATES-Q0000000J</oiEuropeu>
        <organitzacio>Organització de prova</organitzacio>
        <pais>ES</pais>
        <usCertificat>digitalSignature | keyEncipherment</usCertificat>
        <usCertificatExtensio>KeyPurposeId 0:  TLS Web server authentication</usCertificatExtensio>
        <validDesDe>2017-04-13T11:33:04+02:00</validDesDe>
        <validFins>2019-04-13T11:33:04+02:00</validFins>
    </informacioCertificat>
    <resultatValidacioCodi>-1</resultatValidacioCodi>
    <resultatValidacioDescripcio>El certificat ha caducat</resultatValidacioDescripcio>
</resultatValidacio>
