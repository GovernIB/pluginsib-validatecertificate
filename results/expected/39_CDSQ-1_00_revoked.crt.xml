<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<resultatValidacio>
    <informacioCertificat>
        <altresValors/>
        <certificatQualificat>true</certificatQualificat>
        <classificacio>2</classificacio>
        <classificacioEidas>WSA</classificacioEidas>
        <creatAmbUnDispositiuSegur>false</creatAmbUnDispositiuSegur>
        <emissorID>CN=EC-SectorPublic,OU=Serveis Públics de Certificació,O=CONSORCI ADMINISTRACIO OBERTA DE CATALUNYA,C=ES</emissorID>
        <emissorOrganitzacio>CONSORCI ADMINISTRACIO OBERTA DE CATALUNYA</emissorOrganitzacio>
        <numeroSerie>3261199091937075586</numeroSerie>
        <organitzacio>Organització de prova</organitzacio>
        <pais>ES</pais>
        <politica>1.3.6.1.4.1.15096.1.3.2.51.2,0.4.0.194112.1.4</politica>
        <politicaID>MITyC</politicaID>
        <politicaVersio>23</politicaVersio>
        <subject>1.3.6.1.4.1.311.60.2.1.3=#13024553,businessCategory=Government Entity,CN=webtest.aoc.cat,serialNumber=Q0000000J,2.5.4.97=#0C0F56415445532D51303030303030304A,O=Organització de prova,L=Barcelona,C=ES</subject>
        <tipusCertificat>CATCERT SECTORPUBLIC CDSQ-1</tipusCertificat>
        <usCertificat>digitalSignature | keyEncipherment</usCertificat>
        <usCertificatExtensio>KeyPurposeId 0:  TLS Web server authentication</usCertificatExtensio>
        <validDesDe>2017-04-13T11:33:04+02:00</validDesDe>
        <validFins>2019-04-13T11:33:04+02:00</validFins>
    </informacioCertificat>
    <resultatValidacioCodi>3</resultatValidacioCodi>
    <resultatValidacioDescripcio>El certificado esta revocado</resultatValidacioDescripcio>
</resultatValidacio>
